.github/workflows/tests.yml

```diff
@@ -193,3 +193,56 @@ jobs:
         with:
           run: python -m tox -e py37-PyQt5-minimal
         timeout-minutes: 60
+
+  test_conda:
+    name: Test PartSeg conda
+    runs-on: ubuntu-20.04
+    needs: download_data
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create environment.yml
+        shell: bash
+        run: |
+          cat <<EOF > environment.yml
+          name: test
+          channels:
+            - conda-forge
+          dependencies:
+            - conda-smithy
+            - python 3.9
+            - pip
+            - tomlkit
+            - ruamel.yaml
+            - anaconda-client
+            - tox-conda
+          EOF
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: latest
+          channels: conda-forge
+          channel-priority: strict
+          python-version: 3.9
```

Comment:
You need to add `environment-file: environment.yml` here, I think.

Reply:
but what should be the content of such a file?